reference-architecture/vmware-ansible/roles/rhn-subscription/tasks/main.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Is the host already registered?
+  command: "subscription-manager list"
+  register: subscribed
+  ignore_errors: yes
+
+- name: Register host via Activation key
+  redhat_subscription:
+    activationkey: "{{ rhsm_activation_key }}"
+    org_id: "{{ rhsm_org_id }}"
+    state: present
+    pool: "^(60 Day Supported OpenShift Enterprise|OpenShift Enterprise, Standard|OpenShift Enterprise, Premium|Employee)"
```

Comment:
@detiber yea. I needed another provider to work with in regards to moving towards a common location. I will probably begin work on that once i ship the first 3.3 doc. It will definitely help keep some sanity in keeping 1 subscription task in sync rather than 5/6 subscription tasks.